seqr/views/apis/dataset_api.py

```diff
@@ -110,6 +112,37 @@ def add_variants_dataset_handler(request, project_guid):
     Family.bulk_update(
         request.user, {'analysis_status': Family.ANALYSIS_STATUS_ANALYSIS_IN_PROGRESS}, guid__in=family_guids_to_update)
 
+    if project.workspace_name and project.workspace_namespace:
+        if has_analyst_access(project):
+            safe_post_to_slack(
+                SEQR_SLACK_DATA_ALERTS_NOTIFICATION_CHANNEL,
+                """{num_sample} new samples are loaded in https://seqr.broadinstitute.org/project/{guid}/project_page
+                ```{samples}```
+                """.format(
+                    num_sample=len(matched_sample_id_to_sample_record),
+                    guid=project.guid,
+                    samples=[id for id in matched_sample_id_to_sample_record.keys()]
+                ))
+        else:
+            send_html_email(
+                """Hi {user},
+We are following up on your request to load data from AnVIL on {date}.
+We have loaded data from the AnVIL workspace “{namespace}/{name}” to the corresponding seqr project {proj_name}. {num_sample} samples are currently loaded. Let us know if you have any questions.
+Thanks,
+Data Manager from seqr
+""".format(
+                    user=request.user,
```

Comment:
if we have a display name for them (first/ last) we want to use that, otherwise we should use their email. We never want to use the user id

Reply:
I have just found that the `user` here should be the project manager, not the data manager who edits the datasets. We don't know exactly who created the data loading request at this point.  Maybe we should use the project creator as the data loading requestor.

Reply:
The project creator will be one of the project managers, but we want the email top go to all project managers, as I stated in the ticket

Reply:
Okay. How about just greeting the creator and sending it to all the managers' mailboxes?